data_steward/tools/import_rdr_omop.sh

```diff
@@ -85,7 +85,7 @@ for file in $cdm_files; do
   table_name=${filename%.*}
   echo "Importing ${RDR_DATASET}.${table_name}..."
   CLUSTERING_ARGS=""
-  if grep -q person_id resource_files/fields/"${table_name}".json; then
+  if grep -q person_id resource_files/schemas/"${table_name}".json; then
```

Comment:
There are also references to `additional_rdr_tables/` in this file that need to be changed to `rdr`. In these cases it's better to declare `RDR_FIELDS_DIR` so we can change e.g.
`${FIELDS_DIR}/additional_rdr_tables/${table_name}.json` to `${RDR_FIELDS_DIR}/${table_name}.json`.